manifests/s/StefanLobbenmeier/youtube-dl-gui/2.5.4/StefanLobbenmeier.youtube-dl-gui.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: StefanLobbenmeier.youtube-dl-gui
+PackageVersion: 2.5.4
+PackageLocale: en-US
+Publisher: Jelle Glebbeek
+PackageName: Open Video Downloader
+License: GPL-3.0
+LicenseUrl: https://github.com/StefanLobbenmeier/youtube-dl-gui/blob/master/LICENSE
+Copyright: Copyright © 2020-2021 Jelle Glebbeek
+CopyrightUrl: https://github.com/StefanLobbenmeier/youtube-dl-gui/blob/master/LICENSE
+ShortDescription: A cross-platform GUI for youtube-dl made in Electron and node.js
+Description: A cross-platform GUI for youtube-dl made in Electron and node.js
+Moniker: Open Video Downloader

```

```suggestion
PackageUrl: https://github.com/StefanLobbenmeier/youtube-dl-gui
PublisherUrl: https://github.com/StefanLobbenmeier
PublisherSupportUrl: https://github.com/StefanLobbenmeier/youtube-dl-gui/issues
Tags:
- youtube
- downloader
```

If the `PackageName` and the `Moniker` are identical, it is not recommended to add a separate `Moniker` entry.